seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -759,36 +759,39 @@
       "id": "bad88fee1348bd9aedf08816",
       "title": "Link to Internal Sections of a Page with Anchor Elements",
       "description": [
-        "In addition to creating external links, anchor elements can also be used to create internal links, which are links that jump to different sections within a webpage.",
-        "The format is similar to an external link except instead of a URL, you'll use the <code>#</code> and a word to describe the section you want to jump to.",
-        "Here's an example:",
-        "<code>&lt;a href=\"#contact\"&gt;Go to contact section&lt;/a&gt;</code>",
-        "Next, you'll want to create a corresponding anchor link in the HTML where you want the internal link to take users. Instead of using the <code>href</code> attribute, you'll use the <code>name</code> attribute. Here's an example:",
-        "<blockquote>&lt;h1&gt;Contact&lt;/h1&gt;<br>&lt;a name=\"contact\"&gt;&lt;/a&gt;</blockquote>",
-        "Now when users click the \"Go to contact section\" link, they'll be taken to the section of the webpage with the anchor that has the <code>name</code> attribute \"contact\".",
+        "Anchor or <code>a</code> elements can also be used to create internal links to jump to different sections within a webpage.",
+        "To create an internal link, you assign a link's <code>href</code> attribute to a hash symbol <code>#</code> plus the <code>id</code> attribute for an element that you want to internally link to, usually further down the page. You would then need to add the same <code>id</code> attribute to the element you are linking to. An <code>id</code> is an attribute that uniquely describes an element.",
+        "Below is an example of an internal anchor link and its target element:",
+        "<blockquote>&lt;a href=\"#contactsHeader\"&gt;Contacts&lt;/a&gt;<br>...<br>&lt;h2 id=\"contactsHeader\"&gt;Contacts&lt;/h2&gt;</blockquote>",
```

Comment:
This example is great! In the CSS section, all of our class names and ID's are hyphen-delimited. For consistency, I think it would be a good idea to do that here too.